### ISSUE TITLE
What happen if Private key already exist ? 

### ISSUE THREAD

**Issue by MayoudP** [SelfLender/react-native-biometrics#23]
Hello,
I'm wondering, if a user already created a keypair : 
```
  Biometrics.createKeys('Confirm fingerprint in order to activate it')
          .then((publicKey) => {
            console.log(publicKey)
          })
```
The next time he do that, if the key already exist, how does your lib works ? Does it create a new_publicKey and keep the private one on the keychain ? Or does it modify both ? 
Sorry for the small issues, not usefull, but I just would like to know...

**Comment by NappyPirate** [comment]
So currently, this library only maintains one key pair.  If `createKeys` is called again it will overwrite the original keys with a new pair.  It is possible to add functionality that would allow you to maintain multiple pairs, would that functionality be useful to you?

**Comment by MayoudP** [comment]
@NappyPirate Yes, a lot. Since I would like to implement touchID for my app, and implementing a keypair process for the security... But I have to be able to save several private keys (I guess) in case if two differents users connect on the same phone and activate the touchID.

**Comment by Donhv** [comment]
> So currently, this library only maintains one key pair. If `createKeys` is called again it will overwrite the original keys with a new pair. It is possible to add functionality that would allow you to maintain multiple pairs, would that functionality be useful to you?

How about this feature?
